corehq/apps/style/static/style/less/components-3/navs.less

```diff
@@ -12,13 +12,10 @@
       background-color: #dbdbdb;
     }
     &.active {
-      a, a:link, a:visited, a:hover {
+      > a, > a:link, > a:visited, > a:hover {
```

Comment:
I'm not 100% sure this is the right place to make this change so will let @biyeun take a look. g2g from me though